File: cli/linktrail.mjs
#!/usr/bin/env node
// Compatibility alias for the pilot window (DP-0029): the executable is `agentlinkops`.
// One line to stderr, then the same main — stdout stays clean for `--json` readers.
import { main } from './main.js';
process.stderr.write('linktrail: this command is now `agentlinkops`; the old name keeps working during the pilot compatibility window.\n');
process.exitCode = await main();

File: scripts/linktrail.py
#!/usr/bin/env python3
"""Compatibility alias (DP-0029): the CRM helper is now agentlinkops.py beside this file."""
import os
import runpy
import sys

sys.stderr.write("linktrail.py: this helper is now agentlinkops.py; the old name keeps working during the pilot compatibility window.\n")
TARGET = os.path.join(os.path.dirname(os.path.abspath(__file__)), "agentlinkops.py")
sys.argv[0] = TARGET
runpy.run_path(TARGET, run_name="__main__")
